src/iop/filmicrgb.c

```diff
@@ -1758,19 +1758,27 @@ static inline void filmic_chroma_v4(const float *const restrict in, float *const
 
   const int use_output_profile = filmic_v4_prepare_matrices(input_matrix, output_matrix, export_input_matrix,
                                                             export_output_matrix, work_profile, export_profile);
+
+  const float norm_min = exp_tonemapping_v2(0.f, data->grey_source, data->black_source, data->dynamic_range);
+  const float norm_max = exp_tonemapping_v2(1.f, data->grey_source, data->black_source, data->dynamic_range);
```

Comment:
For future-proofing, you may use display white here, instead of constant 1. Same, probably, for the `norm_min` set to display black

Reply:
I don't think so. After all `log_tonemapping_v2()` clamps the input to `[0, 1]` which is in turn the input domain of `filmic_spline()` (whose output domain is `[display_black, display_white]`). The clamping here ensures the clamping in `log_tonemapping_v2()` is a no-op (like the clamping to `[display_black, display_white]` which is done after `filmic_spline()`). I actually think the later clamping can be removed with this change.

Reply:
Right, I was one step after the spline.